### DISCUSSION TITLE
Okçuluk

### DISCUSSION THREAD

**Discussion by tolga0102** [sonsuzus/sonsuzus#366] [Soru Cevap]
Sorun

N okçu hedeflere ok atıyor. 1'den başlayarak numaralandırılmış sonsuz hedef vardır. i'inci okçu ki'nin katı olan tüm hedeflere ateş eder.

Tüm okçuların vurduğu en küçük hedefi bulun.

Giriş

İlk satır bir tamsayı T içerir - toplam test durumu sayısı.

Bunu T testi durumları takip eder. Her test senaryosu aşağıdaki formattadır:

İlk satır doğal bir sayı içerir - N - okçu sayısı. İkinci satır N tane boşlukla ayrılmış tamsayı içerir, burada i'inci tamsayı i'inci okçu için ki'nin değerini gösterir.

Çıktı

Her test durumu için, tüm okçuların vurduğu en küçük hedefi yeni bir satıra yazdırın.

Kısıtlamalar

1 < T < 5
1 < N < 15
1 < ki < 48

Örnek Girdi
1
3
2 3 4

Örnek Çıktı
12

**Comment by tolga0102** [comment]
https://www.hackerearth.com/practice/math/number-theory/basic-number-theory-1/practice-problems/algorithm/archery-1/

**Reply by sonsuzus** [reply]
Örnek girdi çıktı var mı?

**Reply by tolga0102** [reply]
Eklemeyi unutmuşum şimdi ekledim öğretmenim :)
